pages/linux/iftop.md

```diff
@@ -0,0 +1,28 @@
+# iftop
+
+> Show bandwidth usage on an interface by host.
+> More information: <https://linux.die.net/man/8/iftop>.
+
+- Start to show the bandwidth usage(sufficient permission needed):
```

Comment:
Perhaps we could remove the `(sufficient permission needed)` here, and simply prefix the command example with `sudo`? We've taken this approach on other pages.